web/modules/custom/collection/config/install/rdf_entity.mapping.rdf_entity.collection.yml

```diff
@@ -5,11 +5,13 @@ dependencies:
     - rdf_entity.graph.default
     - rdf_entity.graph.draft
     - rdf_entity.rdfentity.collection
-third_party_settings: {  }
 id: rdf_entity.collection
 entity_type_id: rdf_entity
 bundle: collection
 rdf_type: 'http://www.w3.org/ns/adms#AssetRepository'
+graph:
```

Comment:
The whole file change makes no sense and is out-of-scope. It only adds noise to the PR. Please revert.